### ISSUE TITLE
Trigger implicit_getter even if the getter is mutating

### ISSUE THREAD

**Issue by RaphaelRO** [realm/SwiftLint#1589]
In case of a mutating getter, the `get` keyword must be specified to add the `mutating` behavior but this trigger the `implicit_getter` warning.

```swift
struct Counter {
  private var count: Int
  
  init(start: Int = 0) {
    self.count = start
  }
  
  var next: Int? {
    mutating get { // warning: Implicit Getter Violation: Computed read-only properties should avoid using the get keyword. (implicit_getter)
      defer {
        self.count += 1
      }
      
      return self.count
    }
  }
}
```

**Edit:** swiftlint version: 0.19.0

**Comment by marcelofabri** [comment]
Similar to #1309

**Comment by marcelofabri** [comment]
Implemented in #1678.
